### PR TITLE
Check script types in Wasabi CJs

### LINKED ISSUE #6: Sync find tx-s those are not coinjoins
I saw tx with not bech32 script type on the output side. As far as I can remember it was WW1 or WW2.

### PR DESCRIPTION
**Pull request by adamPetho**
Fixes: #6 + updates repo to the latest `nopara/master`

### CODE REVIEW

File: Dumplings.Cli/Program.cs [hunk 1]
```diff
@@ -33,12 +33,12 @@ private static async Task Main(string[] args)
                     var scanner = new Scanner(client);
                     await scanner.ScanAsync(rescan: true);
                 }
-                else if (command == Command.Sync)
+                //else if (command == Command.Sync)
```
Comment (molnard) [inline Dumplings.Cli/Program.cs:36]: What is this?

File: Dumplings/Rpc/VerboseTransactionInfo.cs [hunk 2]
```diff
@@ -61,5 +71,21 @@ private ulong CalculateEquality(IEnumerable<Money> values)
 
             return equality;
         }
+
+        public bool IsWasabi2Cj()
+        {
+            var isNativeSegwitOnly = Inputs.All(x => x.PrevOutput.ScriptPubKey.IsScriptType(ScriptType.P2WPKH)) && Outputs.All(x => x.ScriptPubKey.IsScriptType(ScriptType.P2WPKH)); // Segwit only outputs.
```
Comment (molnard) [inline Dumplings/Rpc/VerboseTransactionInfo.cs:77]: Can you optimize this function? First check what is computationally easy, like the number of inputs and outputs - return immediately if the conditions are not met.

**Review by molnard — CHANGES_REQUESTED** [review]
I stopped the review. Should this be added to outfolder branch?

**Comment by adamPetho** [comment]
+900K line of changes? I have no idea wat happened here.

**Pull request closed without merging**